tensorflow/base-cpu/Dockerfile

```diff
@@ -0,0 +1,89 @@
+FROM tensorflow/tensorflow:latest-py3 as mlbench-worker-base-cpu
```

Comment:
Thanks a lot for the PR. looks great. We should use a fixed-version base image so updates to the TF image don't accidentally break our images